changelog/seqera-enterprise/v25.1.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Seqera Enterprise v25.1
+date: 2025-04-09
+tags: [seqera enterprise]
+---
+
+Seqera Platform Enterprise version 25.1 introduces Studios GA and a number of bug fixes and performance enhancements.
+
+Studios is Seqera's in-platform tool for secure, on-demand, interactive data analysis using containers created from Seqera-managed container template images or your own organization managed custom environments. You only pay for the compute your Studio sessions consume and the compute is adjacent to your data, significantly reducing data transfer costs and wasted time copying data from storage to analysis. This significantly reduces infrastructure management requirements, removes data silos, adheres to strict in-platform role-based access control, and lowers your operational costs. Learn more about Studios in our [documentation](https://docs.seqera.io/platform-enterprise/latest/studios)
+
+## New features and improvements
+
+### Studios
+
+- **Labeled templates**: Labels indicate the status of support (`recommended` or `deprecated`) for a Seqera-managed Studio container template version. Users can migrate a Studio to a new base container template when **Adding as new**.
+- **[Private Studios](https://docs.seqera.io/platform-enterprise/latest/studios/managing#general-config)**: When adding a new Studio, the number of concurrent connections (private vs. all workspace members) can now be defined in **General Config > Collaboration**.
```

Comment:
```suggestion
- **[Private sessions](https://docs.seqera.io/platform-enterprise/latest/studios/managing#general-config)**: When adding a new Studio, the number of concurrent connections (private vs. all workspace members) can now be defined in **General Config > Collaboration**.
```

Reply:
Privacy refers to the session itself (i.e. who can connect to it), not the Studio.